databricks/koalas/tests/test_dataframe.py

```diff
@@ -1453,6 +1453,7 @@ def test_swapaxes(self):
 
         self.assertRaises(AssertionError, lambda: kdf.swapaxes(0, 1, copy=False))
         self.assertRaises(ValueError, lambda: kdf.swapaxes(0, -1))
+        self.assertRaises(TypeError, lambda: kdf.swapaxes())
```

Comment:
Do we need this? Now that it's a kind of syntax error.

Reply:
I see, we don't test for syntax errors. Removed.